mst/aiven-client.md/aiven-client-install.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Aiven Client for Managed Service for TimescaleDB
+excerpt: Install and configure Aiven Command Line tool for fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+
+# Aiven Client for fully managed services
+
+Aiven Client is a command line tool for fully managed services. To use Aiven
+Client you need to first create an authentication token and then configure it to
+connect to your Managed Service for TimescaleDB using the command line.
+
+## Create an authentication token in Managed Service for TimescaleDB
+
+To connect to Managed Service for TimecaleDB using Aiven Client you need to
+create an authentication token.
+
+<procedure>
+
+1.  Login to your [Managed Service for TimescaleDB portal][mst-login].
+1.  Click `User Information` icon in the top left corner.
+1.  In the `User Profile` page, select the `Authentication`tab.
+1.  Click the `Generate Token` button.
+1.  In the `Generate access token` dialog, type a descriptive name for the
+    token and leave the rest of the fields blank.
+1.  Copy the generated authentication token and save it.
+
+</procedure>
+
+## Install and configure Aiven Client and connect to Managed Service for TimescaleDB
+
+The `avn` utility is a Python package, so you can install using pip on a Linux,
+MacOS or a Windows system which have Python installed, using `pip install
+aiven-client` command. For more information about installaing Aiven Client, see
+the [getting started][aiven-github] documentation.
```

Comment:
```suggestion
Aiven Client is provided as a Python package, so you can install it on a Linux,
MacOS, or Windows system using `pip`, if you have already installed Python. 

Use this command: 

```bash
pip install aiven-client
```

For more information about installing the Aiven Client, see the 
[Aiven][aiven-github] documentation.
```

The GH webUI has broken my suggestion, but I'm sure you get the idea.